waspc/data/Generator/templates/server/src/polyfill.ts

```diff
@@ -3,5 +3,8 @@
 
 import { webcrypto } from "node:crypto";
 
-// @ts-ignore
-globalThis.crypto = webcrypto as Crypto;
+if (typeof globalThis.crypto === "undefined") {
```

Comment:
Maybe add a comment about this that this only targets Node20+

Reply:
Yeah, might be useful to have that comment, mostly as an explanation of why this is even needed.